georeference/georeference_ontology.ttl

```diff
@@ -0,0 +1,17 @@
+@prefix dcterms: <http://purl.org/dc/terms/> .
+@prefix georeference: <https://github.com/GAIA-X4PLC-AAD/ontology-management-base/tree/main/georeference/> .
+@prefix gx: <https://w3id.org/gaia-x/core#> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+georeference: a owl:Ontology ;
+    rdfs:label "ontology definition for location and georeference information "@en ;
+    dcterms:contributor "Sebastian Tuttas (3DMS)" ;
+    owl:versionInfo "0.1" .
+
+georeference:Georeference a owl:Class ;
```

Comment:
Question of taste:
GeoReference better than Georeference?
I can live with both.

Reply:
i would leave it at Georeference, it's a common term

Reply:
fine with that